### PR TITLE
Travis ec2

### PR DESCRIPTION
**Pull request by dlukman**
Configuration for test kitchen on EC2.

SELinux causes RSA key rejection when testing on EC2 because the instances have it set to enforcing by deafult. Changing it to permissive solves the issue.

::File.join() cannot receive NilClass arguments so a fix has been added to prevent build failure.

### CODE REVIEW

File: providers/user.rb [hunk 1]
```diff
@@ -124,6 +124,8 @@ def binaries_list(os_family)
   end
 
   if new_resource.ssh_keys
+
+    user_chroot_path = '' if user_chroot_path.nil?
```
Comment (kamaradclimber) [inline providers/user.rb]: OPT: `user_chroot_path ||= ''`

**Comment by kamaradclimber** [comment]
I don't understand where you use SElinux cookbook?

**Comment by dlukman** [comment]
Take a look at the runlist in .kitchen.ec2.yml

**Comment by kamaradclimber** [comment]
ok

**Pull request merged**